static/assets/rpc-methods/request_onchain_funds.json

```diff
@@ -52,7 +52,7 @@
     },
     {
       "name": "amount_out",
-      "description": "The amount expected to be sent by anchor to user at end of transaction",
+      "description": "The amount expected to be sent by anchor to user at end of transaction. This is required if the transaction is associated with a firm quote and for exchange transactions where `asset_in` and `asset_out` are the same.",
```

Comment:
- I think we want to remote "exchange" in this sentence, since transactions where `amout_in.asset` and `amount_out.asset` are the same are "exchange" transactions in the way we use the word in the protocol
- I think we need to change `asset_in` to `amount_in.asset` and `asset_out` for `amount_out.asset`